db/controllers/api.controller.js

```diff
@@ -35,3 +36,20 @@ exports.getArticles = (req, res) => {
     res.status(200).send({ articles: data });
   });
 };
+
+exports.getComments = (req, res) => {
+  const { article_id } = req.params;
+  selectComments(article_id)
+    .then((comments) => {
+      if (comments.length === 0) {
+        return res
```

Comment:
See the comments on testing to work out what to do in this case of there being no comments for a valid and existing article id verses no comments because the valid article id does not exist in the database.